assets/js/templates/sidebar-items.handlebars

```diff
@@ -1,50 +1,65 @@
 <ul id="full-list">
-{{#each nodes as |node nodeId|}}
+  {{#each nodes as |node nodeId|}}
     {{#newGroup ../this node.group}}
       <li class="group">{{node.group}}</li>
     {{/newGroup}}
     <li class="{{#isLocal node.id}}clicked open{{/isLocal}}">
 
-    {{!-- When visiting a module page, the link to this module page
-    in the menu should not link to a new page, instead should link
-    to the top of the page itself. --}}
-    {{#isEmptyNode node}}
-      <a href="{{node.id}}.html" title="{{node.title}}">{{node.title}}</a>
-    {{else}}
-      <a href="{{node.id}}.html" title="{{node.title}}" class="expand">{{node.title}}</a>
+      {{! Nodes added purely for nesting purposes (i.e. don't exist in the source) have no id }}
+      {{#if node.id}}
+        {{! When visiting a module page, the link to this module page
+        in the menu should not link to a new page, instead should link
+        to the top of the page itself. }}
+        <a href="{{node.id}}.html" title="{{node.title}}" {{#hasContent node}}class="expand"{{/hasContent}}>
+          {{#if node.title_truncated}}
+            <span class="nested"></span>{{node.title_truncated}}
```

Comment:
I wonder if we should do this as a CSS class instead?

Reply:
The reason why I think this would be better as a class is because we need to indent all of the content inside the link. In the image below, notice how the inner contents are not aligned, which is weird when expanding multiple groups: 

<img width="285" alt="screen shot 2018-10-23 at 10 15 07" src="https://user-images.githubusercontent.com/9582/47346135-aeca1b80-d6ac-11e8-92f6-b6319583ac23.png">

Reply:
Actually, I would suggest to add the `class="nested"` to the .li itself and not the link. This gives more options to style it later.

Reply:
Yes, that's what I understood from your original comment as that would be the way to get consistent indentation.